### DISCUSSION TITLE
How can I get ahold of the lead developer?

### DISCUSSION THREAD

**Discussion by jseverson** [MerlinCooper/iRacingReplayDirector#106] [General]
Was looking for contact info, but couldn't find it anywhere. Thought I'd try this. Would love to chat with you about this product. thanks!

**Comment by MerlinCooper** [comment]
Well if you have a look at the information provided in the GitHub ‚insight‘ it should be easy to figure out that Vipoo is the originator and my fork is the most active. Furthermore I did get permissions from vipoo that I can accept pull requests in his fork as well.

But not sure whether that makes me to the lead developer.
